public/app/config/collections/all/searchResultFields.js

```diff
@@ -6,11 +6,11 @@
             {
                 key: 'MediaThumbnail',
                 labelText: 'Image',
-                display: 'image'
+                display: 'cover-image'
```

Comment:
I had to separate the `image` (which is an actual `<img>` element) and `cover-image` (which is a `<div>` with `background-size: cover`) display types, and then I've used `cover-image` in search views and `image` in detail views.